upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,18 @@
 == Migration Changes
 
+=== Migrating to 11.0.0
+
+==== LDAP no-import bugfix
+
+In the previous {project_name} version, when LDAP provider was configured with `Import Users` OFF, then it was possible to update the
+user even if some of non-LDAP mapped attributes were changed. This resulted in confusing behaviour, when the attribute appeared to be updated,
+but it was not. In the current version, the update is not allowed to be performed at all instead to tolerate a successful execution
```

Comment:
Not sure what this means "instead to tolerate a successful execution"  Is it needed?
Are you saying that "it will not tolerate a successful execution?"

Reply:
@andymunro For the context, previous behaviour reported success when the expected field / setting wasn't really updated (that's where the name of "lost update" comes from). With the fix, the update is not performed (and the user is informed back, why it wasn't possible to perform it).

The "instead to tolerate a successful execution" is not needed. Maybe better to rephrase the particular sentence as a whole, so it makes more sense.

Reply:
Okay, that sounds like the right idea.  Thanks, @iankko.

Reply:
I've changed this sentence to "In the current version, the update is not allowed to be performed at all if any non-LDAP mapped attributes are changed." Does it sound ok?

Reply:
LGTM AFAICT, thanks Marek!